RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -282,6 +282,8 @@ familiar lookupFamiliarDatafile(string type)
 	//we do not want a fallback here. if no matching familiar is found then do nothing here, a familiar will be automatically set in pre adventure
 	
 	auto_log_debug("lookupFamiliarDatafile is checking for type [" + type + "]");
+	// store what type of fam we are looking for
+	set_property("auto_lastFamiliarLoopupType",type);
```

Comment:
typo? It doesn't matter either way as you spelled it like this everywhere it is used so don't feel like you have to change it.

Reply:
ah it is, will fix